packages/mco/components/protected-applications/list-page.tsx

```diff
@@ -1,158 +1,143 @@
 import * as React from 'react';
+import { ProtectedApplicationViewKind } from '@odf/mco/types/pav';
+import {
+  getApplicationName,
+  getDRPCRef,
+  getPAVDRPolicyName,
+  getPrimaryCluster,
+} from '@odf/mco/utils/pav';
```

Comment:
nits: export pav.ts from index file...
```suggestion
} from '@odf/mco/utils';
```